src/_pytest/main.py

```diff
@@ -368,6 +370,7 @@ def __missing__(self, path):
 class Session(nodes.FSCollector):
     Interrupted = Interrupted
     Failed = Failed
+    _setupstate = None  # type: SetupState
```

Comment:
If it's type is `SetupState` how can it be initialized to `None`? I guess what we want is `_setupstate: SetupState` but that's Python>=3.6 only. So it's good for now.

Reply:
Yeah..